### DISCUSSION TITLE
[Bug] netlify Building Failed

### DISCUSSION THREAD

**Discussion by chalitbkb** [lobehub/lobehub#2675] [Self Hosting] [answered]
### 💻 Operating System

Windows

### 📦 Environment

Other

### 🌐 Browser

Chrome

### 🐛 Bug Description

Encountered issues while deploying on Netlify, but no problems were found when deploying on Vercel.
![image](https://github.com/lobehub/lobe-chat/assets/32437546/c7b5f7a9-7602-44cb-bb05-3db670388f06)

[a comment by an automated account is not shown]

**Comment by alex-radulescu-b1** [comment]
Check https://github.com/lobehub/lobe-chat/issues/1917

**Comment by chalitbkb** [comment]
> Check #1917

I have tried this method. Before asking, I reviewed all the questions and answers. Therefore, using 'pnpm run build' did not work for me. I guess this issue might be due to a memory limit being reached.

**Comment by xingpingcn** [comment, marked as answer]
you can use the AI netlify provides to find where the problem is, for me, setting Environment variables NODE_OPTIONS to "--max_old_space_size=4096" is the solution
